sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/datastore/DatastoreV1.java

```diff
@@ -208,6 +208,14 @@
   static final int DATASTORE_BATCH_UPDATE_LIMIT = 500;
 
   /**
+   * Cloud Datastore has a limit of 10MB per RPC, so we also flush if the total size of mutations
```

Comment:
Can you update the doc here https://github.com/apache/beam/blob/master/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/datastore/DatastoreV1.java#L183 the limitations of writes here (specifically about the possibility of large indexed entities failing).